### PR TITLE
Doc cosmetics

### PR DESCRIPTION
**Pull request by shino**

### CODE REVIEW

File: riak_test/README.md [hunk 1]
```diff
@@ -154,7 +169,7 @@ bag with two riak nodes and two additional bags with one riak node each.
 
 * Your $PATH must have `erl` available.
 * Your $PATH must have a version of Python available that also has
-  access to the Boto S3 libraries.
+  vertualenv tools and access to the Boto S3 libraries.
```
Comment (kuenishi) [inline riak_test/README.md]: Typo

File: src/riak_kv_fs2_backend.erl [hunk 2]
```diff
@@ -20,7 +20,9 @@
 %%
```
Comment (kuenishi) [inline src/riak_kv_fs2_backend.erl]: I think it'd be good to `rm` this module.

File: riak_test/README.md (hunk 1 above)
Comment (shino) [inline riak_test/README.md]: `(／￣▽)／┳━┳‾┫‾┻━┻‾┣‾┳━┳＼(▽￣＼)`

**Comment by shino** [comment]
Thanks for review! Pushed.

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**